kmm/kmm-post-data/src/commonMain/kotlin/com/gchristov/newsfeed/kmmpostdata/RealPostRepository.kt

```diff
@@ -21,29 +20,15 @@ internal class RealPostRepository(
     override suspend fun post(
         postId: String,
         postMetadataFields: String
-    ): DecoratedPost =
+    ): Post =
```

Comment:
The repository returns the `Post` as it comes from the API or transformed to a `domain` model if applicable.